WordPress/Classes/ViewRelated/NUX/Signin2FAViewController.swift

```diff
@@ -75,6 +76,17 @@ import WordPressShared
     // MARK: Configuration Methods
 
 
+    /// Assigns localized strings to various UIControl defined in the storyboard.
+    ///
+    func localizeControls() {
+        verificationCodeField.placeholder = NSLocalizedString("Verification code", comment: "two factor code placeholder")
+
+        let submitButtonTitle = NSLocalizedString("VERIFY", comment: "Title of a button. The text should be uppercase.").localizedUppercaseString
```

Comment:
Should we use lowercase localized strings and `uppercaseStringWithLocale:` them?